tests/jquery.flot.canvaswrapper.Test.js

```diff
@@ -223,6 +223,18 @@ describe('CanvasWrapper', function() {
         expect(elem.childNodes.length).toBe(4);
     });
 
+    it('should evenly space multi line text', function() {
+        var canvas = newCanvas(placeholder);
+        canvas.addText('layerA', 100, 200, '1<br>2<br>3<br>4', 'a');
+        canvas.render();
+
+        var elem = placeholder.find('.a')[0];
+        expect(elem.childNodes[0].getAttribute('dy')).toBe('0em');
```

Comment:
I tried writing this test to use `getBoundingClientRect()` to actually assert that the line spacing is even. But that method does not seem to work well with this SVG content - it returns the same results for every `tspan` element.